src/pitaya/src/ffi/metrics.rs

```diff
@@ -0,0 +1,239 @@
+use async_trait::async_trait;
+use std::{
+    ffi::{c_void, CString},
+    os::raw::c_char,
+};
+
+#[repr(C)]
+pub struct PitayaMetricsOpts {
+    pub namespace: *const c_char,
+    pub subsystem: *const c_char,
+    pub name: *const c_char,
+    pub help: *const c_char,
+    pub variable_labels: *mut *mut c_char,
```

Comment:
In our callbacks we are using `*mut *const c_char` to represent a Vec<String> but here we are using `*mut *mut c_char`. I think it should use `*mut *const c_char` to be consistent with the callbacks.